### ISSUE TITLE
additionalFileMapping: strange behaviour

### ISSUE THREAD

**Issue by Tinyblargon** [leonardssh/vscord#213]
Hi,

I'm trying to get all files ending with `.ansible.yml` to be detected as `ansible`.
Even though I've added the following setting to my config but `.ansible.yml` files keep getting detected as `yml` files.
```json
    "vscord.behaviour.additionalFileMapping": {
        ".ansible.yml": "ansible"
    }
```

When i add the following config all files with the `.something` extension do get detected as `ansible`.

```json
    "vscord.behaviour.additionalFileMapping": {
        ".something": "ansible"
    }
```

Is it possible that [languages.json](https://github.com/leonardssh/vscord/blob/main/src/data/languages.json) takes precedence over `vscord.behaviour.additionalFileMapping`?

**Comment by xhayper** [comment]
Yup, i can do it.

**Comment by xhayper** [comment]
The reason for the `strange behaviour` is because `languages.json` have higher priority than `additionalFileMapping`, i gonna push a commit where `additionalFileMapping` will have higher priority than `languages.json`

**Comment by xhayper** [comment]
Fixed in `v5.1.10`

**Comment by xhayper** [comment]
![image](https://github.com/leonardssh/vscord/assets/46065349/b19d7a1e-b7aa-4fc9-a391-1743890f00b2)
